src/main/java/com/afornalik/ox/board/Board.java

```diff
@@ -0,0 +1,35 @@
+package com.afornalik.ox.board;
+
+import java.util.Map;
+
+public class Board {
+
+    private Map<Integer, BoardField> boardFields;
+    private final int bound;
+
+    public Board(int bound) {
+        this.bound = bound;
+    }
+
+    public int getBound() {
+        return bound;
+    }
+
+    boolean initializeAllField(Map<Integer, BoardField> mapImplementation) {
+        int fieldQuantity = bound * bound;
+        boardFields = mapImplementation;
+        for (int i = 0; i < fieldQuantity; i++) {
+            boardFields.put(i, new BoardField());
+        }
+        return true;
+    }
+
+    BoardField receiveBoardField(int i) throws OutOfBoardException {
+        if (i < 0 || i > (boardFields.size()-1)) {
+            throw new OutOfBoardException();
+        }
+        return boardFields.get(1);
```

Comment:
Only for TTD?

Reply:
Here it looks like it is, but i change it and actually it have some sense because this method return boardField object if it is created before or null references if point doesn't exist but if you aim out of board You get exception, so we have three situations.